Can't pass `dry_run` in there any more.

samcunliffe opened the pull request:
Sorry, somehow this survived and wasn't noticed by the linting. Trivial bug.


I suppose it would have been caught by a full integration test 🙃.

milanmlft reviewed (dismissed):
🍉

samcunliffe commented on prefect/run_omop_es.py:
```diff
@@ -32,7 +32,7 @@ def run_omop_es(
     start_batch: Optional[str] = None,
     extract_dt: Optional[str] = None,
-    dry_run: bool = False,
+    dry_run: bool = False,  # could remove?
```
```suggestion
```

milanmlft commented:
> ![Screenshot 2025-05-28 at 16 54 49](https://private-user-images.githubusercontent.com/1836192/448476159-cbd84529-3848-4b89-aff6-9bd89b3d90c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDg1MDk2NTAsIm5iZiI6MTc0ODUwOTM1MCwicGF0aCI6Ii8xODM2MTkyLzQ0ODQ3NjE1OS1jYmQ4NDUyOS0zODQ4LTRiODktYWZmNi05YmQ4OWIzZDkwYzMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDUyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA1MjlUMDkwMjMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjQ5YjUzOGFmNGMyYTZmOWI1YzgzY2I2ZGNjNzIxYjU2YWI2YWIzZDYxZGYwOTllMjM2M2QwOTkwM2IwOGQ0OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.YFHJz30aypH6LBxeSzLpDLkndLkx0AnKNQX_1vbEfJ4)
> 
> I did not intentionally do this ☝️
> 
> Such noise for a one-liner.

Ooh that might have been me actually. I updated the branch protection rules:
![image](https://github.com/user-attachments/assets/c851b81a-ce27-41a9-9dc5-735bf4910ee2)

but maybe this is a bit too strict 😅

samcunliffe commented:
> > I did not intentionally do this ☝️
> > Such noise for a one-liner.
> 
> Ooh that might have been me actually. I updated the branch protection rules: 
>
> 
> but maybe this is a bit too strict 😅

Let's see how it goes? If I remember correctly, we got annoyed at that on INFORMus and reverted it.

milanmlft commented:
> > > I did not intentionally do this ☝️
> > > Such noise for a one-liner.
> > 
> > 
> > Ooh that might have been me actually. I updated the branch protection rules:
> > but maybe this is a bit too strict 😅
> 
> Let's see how it goes? If I remember correctly, we got annoyed at that on INFORMus and reverted it.

I reverted it already, considering this is still very much under active development and isn't supporting anything critical yet.
